envoyauth/response.go

```diff
@@ -34,14 +34,24 @@ type StopFunc = func()
 // TransactionCloser should be called to abort the transaction
 type TransactionCloser func(ctx context.Context, err error) error
 
+// An Opt allowing externally setting the decision ID
+type Opt func(*EvalResult)
```

Comment:
Nit: do we need to define this?

Reply:
We can remove it. Thanks for the suggestion.